.docker/Dockerfile-template.ckan

```diff
@@ -14,6 +14,7 @@ RUN apk add --no-cache build-base \
 # Install CKAN.
 
 RUN cd $SRC_DIR/ckan \
+    && git config --global --add safe.directory /srv/app/src/ckan \
```

Comment:
Should use $SRC_DIR, see https://github.com/qld-gov-au/ckan-qld-infrastructure/blob/503de20fe159ac9394afd24d83f8c88f0563e42c/.docker/Dockerfile-template.ckan#L20